gk/main.c

```diff
@@ -1798,10 +1862,85 @@ gk_proc(void *arg)
 
 		process_cmds_from_mailbox(instance, gk_conf);
 
-		if (rte_rdtsc() - last_tsc >= bucket_scan_timeout_cycles) {
+		if (rte_rdtsc() - last_scan_tsc >=
+				bucket_scan_timeout_cycles) {
 			gk_flow_tbl_bucket_scan(&bucket_idx,
 				gk_conf->request_timeout_cycles, instance);
-			last_tsc = rte_rdtsc();
+			last_scan_tsc = rte_rdtsc();
+		}
+
+		if (rte_rdtsc() - last_measure_tsc >=
+				basic_measurement_logging_cycles) {
+			float  pkts_num_frac_granted = 0.;
```

Comment:
Following each `float` declaration there are two spaces instead of one before each variable name.